packages/react-components/src/components/Popover/Popover.stories.css

```diff
@@ -21,3 +21,10 @@
   line-height: 20px;
   margin-left: 5px;
 }
+
+.lc-testArea {
```

Comment:
What is `testArea`?

Reply:
And why `lc-` prefix?

Reply:
I know it is a stupid name but could not come up with something better. Basically, it's a div where the component is tested :P

Reply:
fixed